flink-connectors/flink-jdbc/src/main/java/org/apache/flink/api/java/io/jdbc/JdbcExecutionOptions.java

```diff
@@ -0,0 +1,92 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.flink.api.java.io.jdbc;
+
+import org.apache.flink.annotation.PublicEvolving;
+import org.apache.flink.util.Preconditions;
+
+import java.io.Serializable;
+
+/**
+ * JDBC sink batch options.
+ */
+@PublicEvolving
+public class JdbcExecutionOptions implements Serializable {
```

Comment:
It seems that this option is only used for batch writing. `Execution` sounds like it also contains reading/query, because `PreparedStatement` also exposes a `executeQuery` method. 
Shall we rename it to `JdbcWritingOptions` or `JdbcSinkOptions`?

Reply:
`JdbcWritingOptions` doesn't seem to match `delete` queries.
And `JdbcSinkOptions` sounds too generic to me. I'd expect something like JDBC URL inside.

As for `executeQuery`, I don't see why `JdbcExecutionOptions` can't be used for it (batch could mean `limit` then).

Reply:
I don't think batch could mean `limit`. From my understanding, `JdbcExecutionOptions` is just a tuning options which should affect the final result. And the `batchIntervalMs` doesn't make sense for readers. 
But if we dont' have a better name for it, I'm also fine with `JdbcExecutionOptions`.

Reply:
I agree, `JdbcExecutionOptions` isn't ideal, but other names aren't better.